recipes/hyperbox-brain/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "hyperbox-brain" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 41ba26a8b285f48ef5f8e76c6f60fdab43b9ed689b7f2c531e7ade4442a83437
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - numpy
+    - wheel
+    - scikit-learn
+    - pandas
+    - matplotlib-base
+    - joblib
+    - scipy
+    - threadpoolctl
+    - plotly
+  run:
+    - python >=3.6
+    - numpy
+    - scikit-learn
+    - scipy
+    - joblib
+    - threadpoolctl
+    - plotly
+    - pandas
+    - matplotlib-base
+    - setuptools
+
+test:
+  imports:
+    - hbbrain
+    - hbbrain.tests
+    - numpy
+    - pandas
+    - sklearn.datasets
+  requires:
+    - pip
+    - numpy
+    - scikit-learn
+    - pandas
+    - matplotlib-base
+    - pytest
+  commands:
+    - pip check
```

```suggestion
test:
  imports:
    - hbbrain
    - hbbrain.tests
  requires:
    - pip
  commands:
    - pip check
```
This section is for testing whether *your package only* is installed correctly. If the imports and pip check fail without additional requirements, that means your package is broken.

---

@carterbox  I have updated the meta file according to your comments. Thanks.